src/pages/Login/Components/LoginComponent.Styles.ts

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components"
+
+export const LoginComponentLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  min-width: 50%;
+  height: 100%;
+
+  color: ${({ theme }) => theme.colors.white};
+`
+
+export const LoginComponentTitle = styled.h1`
+  font-size: ${({ theme }) => theme.fontSizes.large};
+  text-align: center;
+`
+export const LoginForm = styled.form`
+  min-width: 50%;
+  max-width: 50%;
+`
+
+export const LoginInputContainer = styled.div`
+  height: 28%;
+  position: relative;
+`
+
+export const ErrorMessage = styled.div`
+  position: absolute;
+  bottom: 0;
+  font-size: ${({ theme }) => theme.fontSizes.semiSmall};
+  color: red;
+`
+
+export const ButtonContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  margin-top: 2rem;
+`
+
+interface ButtonProps {
+  action: string
+}
+
+export const Button = styled.button<ButtonProps>`
```

Comment:
[P2] 요 부분에서 interface로 정의하시지 않고 바로 사용하실 수 있을것같아서 리뷰 드립니다!

그리고 보통 styled components를 사용할때 props앞에 $를 많이 붙여준다고 하더라구요
- 요소의 일반 속성과의 차별점을 두기위해
- 혹시 모를 기본 속성과의 충돌을 방지하기 위해!
```suggestion
export const Button = styled.button<{$action: string}>`
```
아마 적용해보시면 해당 요소를 사용하시는 곳에서도 명확하게 구별할 수 있고 좋을 것 같습니다